src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+
+export const PeoplePage = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const { slugName } = useParams();
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then((people) => {
+        setPeopleFromServer(people);
+        setLoading(false);
```

Comment:
don't need to use` setLoading(false)` here, because you have it in finally block